[2024-09-14T13:26:14.149+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-14T13:26:14.164+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgres_hooks_v1.postgres_to_s3 manual__2024-09-14T13:14:43.254219+00:00 [queued]>
[2024-09-14T13:26:14.171+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgres_hooks_v1.postgres_to_s3 manual__2024-09-14T13:14:43.254219+00:00 [queued]>
[2024-09-14T13:26:14.172+0000] {taskinstance.py:2856} INFO - Starting attempt 5 of 8
[2024-09-14T13:26:14.184+0000] {taskinstance.py:2879} INFO - Executing <Task(PythonOperator): postgres_to_s3> on 2024-09-14 13:14:43.254219+00:00
[2024-09-14T13:26:14.194+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_hooks_v1', 'postgres_to_s3', 'manual__2024-09-14T13:14:43.254219+00:00', '--job-id', '735', '--raw', '--subdir', 'DAGS_FOLDER/postgres_hooks_dag.py', '--cfg-path', '/tmp/tmp4wh8s_px']
[2024-09-14T13:26:14.195+0000] {standard_task_runner.py:105} INFO - Job 735: Subtask postgres_to_s3
[2024-09-14T13:26:14.196+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=61740) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-09-14T13:26:14.197+0000] {standard_task_runner.py:72} INFO - Started process 61741 to run task
[2024-09-14T13:26:14.236+0000] {task_command.py:467} INFO - Running <TaskInstance: dag_with_postgres_hooks_v1.postgres_to_s3 manual__2024-09-14T13:14:43.254219+00:00 [running]> on host 01a3c7586b71
[2024-09-14T13:26:14.305+0000] {taskinstance.py:3122} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='ubeyd' AIRFLOW_CTX_DAG_ID='dag_with_postgres_hooks_v1' AIRFLOW_CTX_TASK_ID='postgres_to_s3' AIRFLOW_CTX_EXECUTION_DATE='2024-09-14T13:14:43.254219+00:00' AIRFLOW_CTX_TRY_NUMBER='5' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-09-14T13:14:43.254219+00:00'
[2024-09-14T13:26:14.307+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-14T13:26:14.326+0000] {base.py:84} INFO - Retrieving connection 'postgres_localhost'
[2024-09-14T13:26:14.339+0000] {taskinstance.py:3301} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 767, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 733, in _execute_callable
    return ExecutionCallableRunner(
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 406, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 238, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 256, in execute_callable
    return runner.run(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/dags/postgres_hooks_dag.py", line 22, in postgres_to_s3
    cursor.execute("select * from orders where date <= 20220501")
psycopg2.errors.UndefinedFunction: operator does not exist: date <= integer
LINE 1: select * from orders where date <= 20220501
                                        ^
HINT:  No operator matches the given name and argument types. You might need to add explicit type casts.

[2024-09-14T13:26:14.353+0000] {taskinstance.py:1225} INFO - Marking task as UP_FOR_RETRY. dag_id=dag_with_postgres_hooks_v1, task_id=postgres_to_s3, run_id=manual__2024-09-14T13:14:43.254219+00:00, execution_date=20240914T131443, start_date=20240914T132614, end_date=20240914T132614
[2024-09-14T13:26:14.367+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-09-14T13:26:14.368+0000] {standard_task_runner.py:124} ERROR - Failed to execute job 735 for task postgres_to_s3 (operator does not exist: date <= integer
LINE 1: select * from orders where date <= 20220501
                                        ^
HINT:  No operator matches the given name and argument types. You might need to add explicit type casts.
; 61741)
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py", line 117, in _start_by_fork
    ret = args.func(args, dag=self.dag)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/cli_config.py", line 49, in command
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/cli.py", line 115, in wrapper
    return f(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 483, in task_run
    task_return_code = _run_task_by_selected_method(args, _dag, ti)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 256, in _run_task_by_selected_method
    return _run_raw_task(args, ti)
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 341, in _run_raw_task
    return ti._run_raw_task(
           ^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/session.py", line 97, in wrapper
    return func(*args, session=session, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 2995, in _run_raw_task
    return _run_raw_task(
           ^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 273, in _run_raw_task
    TaskInstance._execute_task_with_callbacks(
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3149, in _execute_task_with_callbacks
    result = self._execute_task(context, task_orig)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3173, in _execute_task
    return _execute_task(self, context, task_orig)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 767, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 733, in _execute_callable
    return ExecutionCallableRunner(
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 406, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 238, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 256, in execute_callable
    return runner.run(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/dags/postgres_hooks_dag.py", line 22, in postgres_to_s3
    cursor.execute("select * from orders where date <= 20220501")
psycopg2.errors.UndefinedFunction: operator does not exist: date <= integer
LINE 1: select * from orders where date <= 20220501
                                        ^
HINT:  No operator matches the given name and argument types. You might need to add explicit type casts.

[2024-09-14T13:26:14.411+0000] {local_task_job_runner.py:261} INFO - Task exited with return code 1
[2024-09-14T13:26:14.428+0000] {taskinstance.py:3891} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-09-14T13:26:14.432+0000] {local_task_job_runner.py:240} INFO - ::endgroup::
